.github/workflows/stale.yml

```
@@ -28,10 +28,6 @@ jobs:
           days-before-close: 5
```

```suggestion
          days-before-issue-stale: -1
          days-before-issue-close: -1
          days-before-pr-stale: 30
          days-before-pr-close: 5
```

@soimkim, 27-28번 줄의 설정을 override해주시면 됩니다. suggestion에서는 27번 줄이 포함되지 않았는데 수정 반영 부탁드리겠습니다:)